setup-homebrew/action.yml

```diff
@@ -14,20 +14,17 @@ inputs:
     required: false
     default: false
   token:
-    description: Token to be used for GitHub authentication (if using private repositories)
+    description: Token to be used for GitHub authentication (if using private repositories). This token will persist through other steps for the duration of the job.
```

Comment:
```suggestion
    description: Token to be used for GitHub authentication (if using private repositories). This token will only persist through other steps for the duration of the job.
```

Maybe?

Reply:
Possibly, but the tone of this is intended to be more of a security warning, to let the user know that the token could be read by subsequent steps (in case there's any untrusted code being run), rather than it being scoped to just setup-homebrew.

Reply:
1) Yes, with an opt-out (`persist-credentials`). Though they also apply the token by default unlike us.
2) action.yml states `The post-job step removes the PAT.`. The better description is at the near the start of the README: https://github.com/actions/checkout/blob/main/README.md

Reply:
Fine with me either way. Let's emulate what they do as much as possible, I reckon, because we're sort of a replacement for that for taps.